Source/Core/UICommon/ResourcePack/Manager.cpp

```diff
@@ -124,9 +135,8 @@ bool Add(const std::string& path, int offset)
 
   file.Save(packs_path);
 
-  packs.insert(packs.begin() + offset, std::move(pack));
-
-  return true;
+  auto it = packs.insert(packs.begin() + offset, std::move(pack));
+  return &*it;
```

Comment:
There's only one caller that checks the return value, and it will always be packs.back() from there, so this probably isn't necessary.

On the other hand I think Manager.cpp could use a lot of refactoring beyond the scope of this PR, and a clean solution would probably depend on that, so go ahead and do whatever seems best.

Reply:
I think I'll leave this as-is.